### ISSUE TITLE
Playback of replay chat bugged

### ISSUE THREAD

**Issue by haxplayer** [haxball/haxball-issues#587]
If you watch a replay of a hbr2 and skip through, it doesn't show the chat history of the parts you skipped.
Also if you skip back earlier, the chat still remains and will get repeated if you play back the same part again.

**Comment by teisseire117** [comment]
It's not a bug it's how the replay works

**Comment by haxplayer** [comment]
> It's not a bug it's how the replay works

It is clearly a bug and not expected behaviour for a replay player.
If I want to find something that someone said, I would have to watch the whole replay for the chat to show up in the player. If I skip forward then I might miss the chat text because it doesn't show up if its not played back.

**Comment by Skarmunds** [comment]
I noticed it also a lot earlier. I don't remember this behaviour with Flash replays. Could it be related with a move to HTML5? I doubt it though and think it's a bug.

**Comment by teisseire117** [comment]
> > It's not a bug it's how the replay works
> 
> It is clearly a bug and not expected behaviour for a replay player.
> If I want to find something that someone said, I would have to watch the whole replay for the chat to show up in the player. If I skip forward then I might miss the chat text because it doesn't show up if its not played back.

This is how the replay system is, basro said it himself.
It's clearly not a bug. You don't seem to know the definition of a 'bug'

**Comment by haxplayer** [comment]
> This is how the replay system is, basro said it himself.
> It's clearly not a bug. You don't seem to know the definition of a 'bug'

I don't think you understand what I'm saying or you don't use the replay feature.
I will try to explain it better if you are too lazy to try and replicate the bug yourself.

It's like if you skip to the end of the game, the score still says 0-0 even though multiple goals were scored during the game. Only instead of goals it's the chat display.

The reason I believe it's a bug is because it wasn't like this in the flash version and it doesn't make sense if it was intended this way. Please show me where basro said it's as intended instead of hijacking this issue with no explanations in your posts.

**Comment by haxplayer** [comment]
> This is how the replay system is, basro said it himself.
> It's clearly not a bug. You don't seem to know the definition of a 'bug'

I don't think you understand what I'm saying or you don't use the replay feature.
I will try and explain it better since you are too lazy to try and replicate the bug yourself.

It's like if you skip to the end of the game but the score still shows 0-0 even though there were multiple goals scored during the game. Instead of goals it's the chat display.

I believe it's a bug because it wasn't like this in the flash version and it doesn't make sense if it was intended.
Please show me where basro said it was intended instead of hijacking this issue without any explanation or reasoning in your posts.
